### DISCUSSION TITLE
Feature: Audible alerts when approaching a marker

### DISCUSSION THREAD

**Discussion by pricewatermouse** [osmandapp/OsmAnd#17372] [Ideas]
As a user of OsmAnd, I want to be notified audibly when I am approaching a marker within a given radius, so that I can be alerted about my proximity to selected locations without using navigation mode.

### Acceptance Criteria:

1. The user should have the ability to set one or two different radius values, triggering alerts at different distances from the marker.
2. The alerts should include both sound and/or vibration, ensuring they can grab the user's attention in different environments (e.g., noisy traffic areas, etc).
3. The feature should be configurable to enable or disable notifications based on user preferences.
4. The radiuses should be visible on the map as a circular areas around the markers.
5. The feature should work even when the app is not in navigation mode.

### Scenario 1: Long-distance bus travel in Asia

- Given that I am traveling in a long-distance bus in Asia and I have set a marker for my desired destination on OsmAnd and I have specified two radiuses, a larger one ("First Alert", 3 km) and a smaller one ("Final Alert", 300 m).
- When I approach the larger radius around the marker then OsmAnd should issue a signal, such as a sound and/or vibration, so that I can be alerted in advance and prepare to disembark.
- When I enter the smaller radius around the marker OsmAnd should issue a second audible signal, indicating that I am very close to my destination and I can request a stop.

### Scenario 2: Pedestrian walk along a trail

- Given that I am walking along a trail and I have set a marker for a specific turn/junction/waypoint on OsmAnd and I have specified the radius (50 m) for the alert.
- When I approach the radius around the marker, OsmAnd should issue a signal, such as a sound and/or vibration, so that I can be alerted about an important point.

In both scenarios, the audible signals should serve as effective notifications to alert me about my proximity to the desired locations, allowing me to stay informed without relying on constant navigation guidance. I would rather prefer tone alerts than tts.

**Comment by netimen** [comment]
Excellent idea! Also waiting for this. Very handy when traveling by bus

**Comment by TangoFan** [comment]
I too would like such a feature.  I like the idea of an alert when not actively navigating.
